### PR TITLE
[Feat] Add event manage options

### PR DESCRIPTION
**Pull request by Chiranjibichapagain**
desktop
mob

### CODE REVIEW

File: client/src/components/Event/Event.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@
   box-sizing: border-box;
   display: flex;
   flex-direction: column;
+  position: relative;
```
Comment (MikeVedsted) [inline client/src/components/Event/Event.scss:8]: Why are you adding `position: relative;` ? It doesn't look like you are actually using it

File: client/src/components/Event/index.tsx [hunk 2]
```diff
@@ -24,7 +26,11 @@ const Event = ({
   handleAddRequest
 }: EventType) => {
   const [details, setDetails] = useState(false)
+  const [showManageOptions, setShowManageOptions] = useState(false)
   const [users] = useUserDisplay()
+  const [cookies, setCookies] = useCookies(['user'])
+
+  const { user_id } = cookies.user || ''
```
Comment (MikeVedsted) [inline client/src/components/Event/index.tsx:33]: remove empty line 32

File: client/src/components/Event/index.tsx [hunk 3]
```diff
@@ -33,13 +39,42 @@ const Event = ({
...
+  const endEvent = () => {
+    setShowManageOptions(false)
+  }
+
+  const editEvent = () => {
+    setShowManageOptions(false)
+  }
+
+  console.log('user_id', user_id, 'creator_id', eventCreator?.user_id)
+
```
Comment (MikeVedsted) [inline client/src/components/Event/index.tsx:55]: remove unneeded console.log

File: client/src/components/Event/index.tsx [hunk 4]
```diff
@@ -33,13 +39,42 @@ const Event = ({
...
+    setShowManageOptions(false)
+  }
+
+  const endEvent = () => {
+    setShowManageOptions(false)
+  }
+
+  const editEvent = () => {
+    setShowManageOptions(false)
+  }
```
Comment (MikeVedsted) [inline client/src/components/Event/index.tsx:52]: Are you going to create these functions or create an issue for them? 😉

**Review by MikeVedsted — APPROVED** [review]

File: client/src/components/Event/Event.scss (hunk 1 above)
Comment (Chiranjibichapagain) [reply client/src/components/Event/Event.scss:8]: Yes, it is being used.  I needed to display the dropdown absolute but relative to the parent div (event).

File: client/src/components/Event/index.tsx (hunk 4 above)
Comment (Chiranjibichapagain) [reply client/src/components/Event/index.tsx:52]: Now they just close the dropdown, but when other components are ready like edit page participants modal etc. then we can add them.

Comment (MikeVedsted) [reply client/src/components/Event/index.tsx:51]: If you are now making them now, please create an issue for them

**Review by Rostislav013 — APPROVED** [review]

**Comment by Rostislav013** [comment]
does options close itself if I click outside of it?

**Pull request merged**